### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by Vishruta-Patil**
Pages - 
1. Sign in
2. Login

- Added protected routes
- Added modal for home page

### CODE REVIEW

File: src/components/Common/VideoCard/index.js [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
 
-const VideoCard = ({video}) => {
+const VideoCard = ({ video }) => {
+  const [modalVisibility, setModalVisibility] = useState(false);
   return (
     <section className="video-card">
       <div className="img-container flex-center">
-        <img className="video-img" src={video.img_url} alt=""/>
+        <img className="video-img" src={video.img_url} alt="" />
       </div>
```
Comment (prathmeshjagtap) [inline src/components/Common/VideoCard/index.js:11]: add something in alt tag

File: src/pages/auth/login.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      <button className="hero-btn" onClick={loginHandler}>
+        Login
+      </button>
+      <button className="outline-btn" onClick={guestCredentailsHandler}>
+        Login With Guest Credentials
+      </button>
+      <p className="gen-msg">
+        Dont have an account,{" "}
+        <Link className="primary-color" to="/signin">
+          {" "}
```
Comment (prathmeshjagtap) [inline src/pages/auth/login.js:54]: you can remove this

**Review by prathmeshjagtap — APPROVED** [review]
I liked that you have made separate files to make API call , rest everything  is good

**Review by parthgabani777 — APPROVED** [review]
code looks well structured.

**Pull request merged**